src/test/resources/pipelines/githubTest.groovy

```diff
@@ -144,13 +144,20 @@ node ('zowe-jenkins-agent') {
                 echo "[GITHUB_TEST] getting correct user permission from pull request successfully"
             }
         }
-        
+
+        //posting a comment on PR
+        def commentPostedTime = github.postComment(prId,"Hello, this is a test comment")
```

Comment:
Could we use the test string with several special characters (including double quotations, backslashes, line feeds, carriage returns, dollar signs) here?